python/pyspark/pandas/tests/test_dataframe.py

```diff
@@ -6000,6 +6000,14 @@ def test_combine_first(self):
             expected_pdf = pd.DataFrame({"A": [None, 0], "B": [4.0, 1.0], "C": [3, 3]})
             self.assert_eq(expected_pdf, psdf1.combine_first(psdf2))
 
+    def test_multi_index_dtypes(self):
+        arrays = [[1, 1, 2, 2], ["red", "blue", "red", "blue"]]
+        idx = pd.MultiIndex.from_arrays(arrays, names=("number", "color"))
+        pdf = pd.DataFrame(np.random.randn(4, 5), idx)
+        psdf = ps.from_pandas(pdf)
+
+        self.assert_eq(psdf.index.dtypes, pdf.index.dtypes)
```

Comment:
nit: Can we just compare with MultiIndex itself ??

For example,

```python
        arrays = [[1, 1, 2, 2], ["red", "blue", "red", "blue"]]
        pmidx = pd.MultiIndex.from_arrays(arrays, names=("number", "color"))
        psmidx = ps.from_pandas(pmidx)

        self.assert_eq(psmidx.dtypes, pmidx.dtypes)
```

Reply:
thanks, updated.